sound/soc/intel/common/soc-acpi-intel-lnl-match.c

```diff
@@ -575,6 +576,7 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_lnl_sdw_machines[] = {
 		.links = lnl_cs42l43_l0,
 		.drv_name = "sof_sdw",
 		.sof_tplg_filename = "sof-lnl-cs42l43-l0.tplg",
+		.get_function_tplg_files = sof_sdw_get_tplg_files,
```

Comment:
the commit messages for the match files also needs update s/get_tplg_files/get_function_tplg_files

Reply:
s/get_tplg_function_files/get_function_tplg_files